packages/contracts/deploy/009-Colosseum.ts

```diff
@@ -21,7 +21,10 @@ const deployFn: DeployFunction = async (hre) => {
       l2OutputOracleProxyAddress,
       zkVerifierAddress,
       hre.deployConfig.l2OutputOracleSubmissionInterval,
-      hre.deployConfig.colosseumChallengeTimeout,
+      hre.deployConfig.colosseumBisectionTimeout,
+      hre.deployConfig.colosseumProvingTimeout,
```

Comment:
are there any reason to omit `CHAIN_ID`, `GUARDIAN`?

Reply:
Cool! BTW, I see you used `portalGuardian` for the security council address, is this temporary until the security council is decided? Or is it possible that guardian for `KromaPortal` and security council for `Colosseum` are the same?

Reply:
Currently, I think of portal guardian and security council as the same group.